_rules/image-no-text-0va7u6.md

```diff
@@ -0,0 +1,114 @@
+---
+id: 0va7u6
+name: HTML graphics contain no text
+rule_type: atomic
+description: |
+  This rule checks that images of text are not used
+accessibility_requirements:
+  wcag20:1.4.5: # Images of Text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Language
+acknowledgments:
+  authors:
+    - Carlos Duarte
+  images:
+    - Letter posted from Dulverton (Somerset, England) to Bristol (England) in 1894. Released into the public domain by Adrian Pingstone.
+---
+
+## Applicability
+
+This rule applies to any `img` element that is [visible][] except if the `src` [attribute value][] references an SVG document.
+
+## Expectation
+
+The image resource referenced by the `src` [attribute value][] of each target element either does not contain text expressing anything in a [human language][] or it is [essential][] that the text is rendered with that specific presentation.
+
+## Assumptions
+
+The text is the most significant content in the image. If there is text in the image, but it is not the most significant content, this rule might fail but the success criterion might still be satisfied.
```

Comment:
Not sure why this is an assumption rather than part of the Expectation. Do you feel it's ambiguous? (it certainly is subjective).
I guess this is to exclude, say, image of a street with street signs or shop signs showing text. I'm a bit afraid this is going to exclude a ton of images from the rule (i.e. that it is a very strong assumption that often won't hold).

Reply:
Yes, I do feel "most significant content" is ambiguous. That's why I moved this to an assumption.

It would be nice to have input on this from other reviewers...

Reply:
The WCAG definition for "image of text" already excludes "text that is part of a picture that contains significant other visual content", so I'm not sure we need to have this?

Reply:
But we don't link to the WCAG definition of "image of text" so we need to have this here (or worked into the expectation).
And I don't think we could use it, because, at least for me, the part about "significant other visual content" is ambiguous. If you consider the example given in the definition, an image of "a person's name on a name tag in a photograph", assuming that the person is also on the photograph, I wouldn't consider it to be an image of text because there would be other significant visual content on the image.. So, I guess, this is ambiguous...

Reply:
My main concern would not be the "how much significant" but the "is this part of the image significant". As I said in my previous reply, the example of "image of text" in the WCAG definition is not something that I would consider an image of text (it's true that the example needs context that is not provided, so I had to assume the context), so I have to conclude that the "significant other visual content" is being ambiguously interpreted. 

Nevertheless, I seem to be the only one having this doubt, so I can give it a try and include this in the expectation.